Makefile.am

```diff
@@ -14,6 +14,7 @@ DISTCHECK_CONFIGURE_FLAGS = \
 	--with-audit \
 	--with-selinux \
 	--with-imaevm \
+	--with-fsverity \
```

Comment:
Sorry for the silence, I've been on a vacation. We need to get this past the CI to consider merging.

I'd suggest taking this out of the initial autofoo commit, and instead add a new "Enable fsverity in CI" commit to the end of the series that adds this *and* fsverity-utils-devel to ci/Dockerfile along with the other build dependencies.

Reply:
Hi Panu,

No worries, hope you had a good break! I have been swamped with another project the last month, but just made the changes you suggested and pushed them in. I think it worked, at least it looks like it all passed. It's the first time I ever dealt with Docker, so let me know if I broke something.

If there is anything else you think is needed, please let me know.

Jes